core/chains/evm/config/toml/defaults/Berachain_Testnet.toml

```diff
@@ -0,0 +1,18 @@
+ChainID = '80084'
+# finality_depth: instant
+FinalityDepth = 10 
+# block_time: 5s 
+LogPollInterval = '10s' 
+
+FinalityTagEnabled = true
+
+[GasEstimator]
+EIP1559DynamicFees = true
+Mode = 'FeeHistory'
+
+[GasEstimator.FeeHistory]
+# block_time was: 5s
```

Comment:
Missing LogPollInterval?

Reply:
& NoNewFinalizedHeadsThreshold?

Reply:
`LogPollInterval` it's present
`NoNewFinalizedHeadsThreshold` we are using the default value from fallback (which is 0), but I'll add it for consistency to 10m.